static/header.less

```diff
@@ -178,16 +178,14 @@
     @media screen and (max-width: 399px){
         display: none;
     }
-    padding: 8px 5px 10px;
     margin: 0 0 0 @gutter;
     list-style: none;
     justify-content: center;
     align-items: center;
+    padding: 0 5px;
```

Comment:
I think letting flexbox vertically center it (instead of using padding to adjust it) looks better.